infoschema/tables.go

```diff
@@ -1305,15 +1316,49 @@ func GetTiKVServerInfo(ctx sessionctx.Context) ([]ServerInfo, error) {
 	}
 	var servers []ServerInfo
 	for _, store := range stores {
-		tp := tikv.GetStoreTypeByMeta(store).Name()
-		servers = append(servers, ServerInfo{
-			ServerType:     tp,
-			Address:        store.Address,
-			StatusAddr:     store.StatusAddress,
-			Version:        store.Version,
-			GitHash:        store.GitHash,
-			StartTimestamp: store.StartTimestamp,
-		})
+		if !isTiFlashStore(store) {
+			tp := tikv.GetStoreTypeByMeta(store).Name()
+			servers = append(servers, ServerInfo{
+				ServerType:     tp,
+				Address:        store.Address,
+				StatusAddr:     store.StatusAddress,
+				Version:        store.Version,
+				GitHash:        store.GitHash,
+				StartTimestamp: store.StartTimestamp,
+			})
+		}
+	}
+	return servers, nil
+}
+
+// GetTiFlashServerInfo returns all TiFlash nodes information of cluster
+func GetTiFlashServerInfo(ctx sessionctx.Context) ([]ServerInfo, error) {
```

Comment:
Seems we can eliminate the duplicated code?

Reply:
Fixed